docs/diagrams/EditSequenceDiagram.puml

```diff
@@ -0,0 +1,69 @@
+@startuml
+!include style.puml
+
+box Logic LOGIC_COLOR_T1
+participant ":LogicManager" as LogicManager LOGIC_COLOR
+participant ":InternBuddyParser" as InternBuddyParser LOGIC_COLOR
+participant ":EditCommandParser" as EditCommandParser LOGIC_COLOR
+participant "e:EditCommand" as EditCommand LOGIC_COLOR
+participant ":CommandResult" as CommandResult LOGIC_COLOR
+end box
+
+box Model MODEL_COLOR_T1
+participant ":Model" as Model MODEL_COLOR
+end box
+
+[-> LogicManager : execute("edit 2 n/Google")
+activate LogicManager
+
+LogicManager -> InternBuddyParser : parseCommand("edit 2 n/Google")
+activate InternBuddyParser
+
+create EditCommandParser
+InternBuddyParser -> EditCommandParser
+activate EditCommandParser
+
+EditCommandParser --> InternBuddyParser
+deactivate EditCommandParser
+
+InternBuddyParser -> EditCommandParser : parse("2 n/Google")
+activate EditCommandParser
+
+create EditCommand
+EditCommandParser -> EditCommand
+activate EditCommand
+
+EditCommand --> EditCommandParser : d
```

Comment:
Are we allowed to label what `d` is, since a reader would not know what it refers to? Or is it supposed to make it simple haha

Reply:
I think they labelled it previously as d for DeleteCommand so they used d. So if he renamed EditCommand to e probably should use e. But about this, do we need to use labels? I'm not sure how they originally implemented the delete command, but if it's the same as how it is in InternBuddy, DeleteCommandParser simply returns a new instance of DeleteCommand without assigning it to any variable. So was the label of d in the original DeleteCommand sequence diagram solely for labelling purposes only? And if so do we continue labelling in our sequence diagrams?

Reply:
i changed it to e:editcommand as suggested by intellj

Reply:
i think e is ok